Automattic/wp-calypso
# Site Transfers: Clean up feature flag for site transfers

Closes https://github.com/Automattic/dotcom-forge/issues/2907

## Proposed Changes

This PR removes the `yolo/site-transfers-i1` feature flag from the codebase now that we have launched the site transfers feature successfully.

## Testing Instructions

* Check the Calypso Live link
* Navigate to **Settings > General > Transfer your site**
* Ensure that the option to transfer the site is present on both Simple and Atomic sites

## Pre-merge Checklist
